Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2021-05-06
Initial Package Version: 3.0.12
Upstream Status: Unknown
Origin: https://767796.bugs.gentoo.org/attachment.cgi?id=685002&action=diff&format=raw&headers=1
Description: Fixes the build with gcc11.

--- a/modules/demux/adaptive/playlist/SegmentInformation.cpp	
+++ a/modules/demux/adaptive/playlist/SegmentInformation.cpp	
@@ -34,6 +34,7 @@ 
 
 #include <algorithm>
 #include <cassert>
+#include <limits>
 
 using namespace adaptive::playlist;
 
--- a/modules/demux/hls/playlist/Parser.cpp	
+++ a/modules/demux/hls/playlist/Parser.cpp	
@@ -42,6 +42,7 @@ 
 #include <map>
 #include <cctype>
 #include <algorithm>
+#include <limits>
 
 using namespace adaptive;
 using namespace adaptive::playlist;
